src/musort.py

```diff
@@ -127,6 +127,10 @@ def rename_music(self):
                 rename.append(self.separator)
             rename.pop()
             rename = ''.join(rename)+ext
+            """Replacing forbidden path characters in UNIX and Windows with underscores"""
+            for forbidden_symbol in ['\\', '/', '|', '*', '<', '>', '"', '?']:
+                if forbidden_symbol in rename:
+                    rename = rename.replace(forbidden_symbol, "_")
```

Comment:
It would be useful to print an error log. Something like `logging.error("Found forbidden symbol in metadata, replaced symbol with _")` would do fine.